testing/bdd/conftest.py

```diff
@@ -133,12 +133,26 @@ def gen_second_document_fixture(test_context, template_dict, template_file_two):
 
 @when('the EBU-TT-Live document is denested')
 def convert_to_ebuttd(test_context):
+    print('denester input')
+    print(test_context["document"].get_xml())
     test_context["document"] = DenesterNode.denest(test_context["document"])
+    print('denester output')
+    print(test_context["document"].get_xml())
+
+@then('the EBU-TT-Live document is valid')
+def then_ebutt3_document_valid(test_context):
+    ebutt3_document = test_context['document']
+    ebutt3_document.validate()
+    print('valid EBU-TT-Live document:')
+    print(ebutt3_document.get_xml())
+    assert isinstance(ebutt3_document, EBUTT3Document)
```

Comment:
I assumed there was already a step that validated the EBU-TT-Live document already, but `@then('document is valid')` does things quite differently. This is fine :)